docs/setup/getting.md

```diff
@@ -18,3 +18,19 @@ libcu++ is an open source project developed on GitHub, which is where you'll
   find the latest versions and the development branch.
 Our GitHub repository is [github.com/nvidia/libcudacxx](https://github.com/nvidia/libcudacxx).
 
+You can clone the repository via
+```bash
+git clone -c core.symlinks=true https://github.com/nvidia/libcudacxx
+```
+
+After cloning the repository, one needs to initialize the llvm submodule:
+```bash
+git submodule init 
```

Comment:
For whatever reasons, it is currently still not possible to sparsely checkout a submodule. So if a user wants to avoid pulling in all of llvm-project, then they will need to use this slightlöy cumbersome workaround